packages/react-composites/src/composites/CallComposite/CallComposite.test.tsx

```diff
@@ -109,11 +109,4 @@ describe('CallComposite device permission test for call readiness opted in/opted
     expect(audioDevicePermissionRequests).toBe(1);
     expect(videoDevicePermissionRequests).toBe(1);
   });
-
-  /* @conditional-compile-remove(call-readiness) */
-  test('Audio and video device permission should still be requested when callreadiness option opted in', async () => {
-    mount(<CallComposite adapter={adapter} options={{ callReadinessOptedIn: true }} />);
```

Comment:
Should this test be removed? or should we be changing it to make sure the permissions are requested with CallReadiness on?

Reply:
It's identical to the one above it so I removed it